packages/katello/katello/katello-backup

```diff
@@ -1,18 +1,5 @@
 #!/usr/bin/env ruby
```

Comment:
How about a simple shell script for this? I'd also `exit 1` so cronjobs start to fail.